paasta_tools/check_marathon_services_replication.py

```diff
@@ -301,18 +164,13 @@ def check_service_replication(
     """
     expected_count = instance_config.get_instances()
     log.info("Expecting %d total tasks for %s" % (expected_count, instance_config.job_id))
-    proxy_port = marathon_tools.get_proxy_port_for_instance(
-        name=instance_config.service,
-        instance=instance_config.instance,
-        cluster=instance_config.cluster,
-        soa_dir=instance_config.soa_dir,
-    )
+    proxy_port = get_proxy_port_for_instance(instance_config, soa_dir=instance_config.soa_dir)
```

Comment:
Is soa_dir an attribute of our instance configs? Cause that is cool.
If so, how about just always extract that from instance_config inside this function and simplify the calling args?
We'll never need to read an instance_config from one place and use a different soa_dir to lookup the port elsewhere,

Reply:
oh good point, I don't see why not!